### PR TITLE
Feature/step progress 2

### PR DESCRIPTION
**Pull request by duguowei1000**
just the steps UI part

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ import {
   getVariableDebtBalances,
 } from '../utils/balances'
 import { ethers } from 'ethers'
+import Step_progess from './stepprogress/step_progress'
```
Comment (andyrobert3) [inline pages/index.tsx:20]: We should be consistent in naming style, camel case, pascal case or snake case

Suggest sticking with pascal case for components

File: pages/stepprogress/step_progress.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import StepNavigation from "./stepNavigation";
+
+function Step_progess() {
+
+  const labelArray = ['Step 1', 'Step 2', 'Step 3', 'Step 4']
+  const [currentStep, updateCurrentStep] = useState(1);
+
+  function updateStep(step) {
+    updateCurrentStep(step);
+  }
```
Comment (andyrobert3) [inline pages/stepprogress/step_progress.tsx:11]: why do we need the function here?

File: pages/stepprogress/step_progress.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import StepNavigation from "./stepNavigation";
+
+function Step_progess() {
+
+  const labelArray = ['Step 1', 'Step 2', 'Step 3', 'Step 4']
```
Comment (andyrobert3) [inline pages/stepprogress/step_progress.tsx:6]: should make this into an enum

File: pages/stepprogress/step_progress.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+  function updateStep(step) {
+    updateCurrentStep(step);
+  }
+
+  return (
+    <div className="App">
+      <StepNavigation labelArray={labelArray} currentStep={currentStep} updateStep={updateStep}></StepNavigation>
+      <p>Selected Step: {currentStep}</p>
+      <button className="primaryButton" disabled={currentStep === 1} onClick={() => updateStep(currentStep - 1)}>Previous Step</button>
```
Comment (andyrobert3) [inline pages/stepprogress/step_progress.tsx:17]: we want to remove "previous step" here?

**Review by andyrobert3 — APPROVED** [review]

File: pages/stepprogress/step_progress.tsx (hunk 2 above)
Comment (duguowei1000) [reply pages/stepprogress/step_progress.tsx:11]: yes we can remove this if we dont intend to have previous step

**Pull request merged**